src/elm/Page/Dashboard.elm

```diff
@@ -122,15 +129,19 @@ view loggedIn model =
 
         ( Loaded communities, LoggedIn.Loaded profile ) ->
             div [ class "container mx-auto" ]
-                [ div [ class "text-gray-600 text-2xl font-light flex mt-6 mb-4" ]
+                [
+                 button
+                     [ onClick (h.showFeedback { message = "Hi!", success = False }) ]
+                     [ text "hello" ]
+                , div [ class "text-gray-600 text-2xl font-light flex mt-6 mb-4" ]
                     [ text (t "menu.my_communities")
                     , div [ class "text-indigo-500 ml-2 font-medium" ]
                         [ text (profile.userName |> Maybe.withDefault (Eos.nameToString profile.account))
                         ]
                     ]
-                , viewBalances loggedIn communities
-                , viewVerifications loggedIn.shared model
-                , viewSections loggedIn model
+                , viewBalances loggedIn communities |> Html.map h.tagger
+                , viewVerifications loggedIn.shared model |> Html.map h.tagger
+                , viewSections loggedIn model |> Html.map h.tagger
```

Comment:
Hmm mapping the Dashboard messages seems very weird, it should be the other way around

Reply:
I mean, local messages shouldn't need any mapping, outside messages might need. Also the feedback should be global, that is, something outside of the page itself. Like the menu, for example. Its global and don't live in the individual page